app/assets/stylesheets/style.scss

```diff
@@ -370,13 +370,13 @@ label:hover {
 }
 
 .center_spinner {
- width: 100px;
- height: 60px;
- position: absolute;
- left: 50%;
- top: 50%;
- margin-left: -50px;
- margin-top: -30px;
+  width: 100px;
```

Comment:
Properties should be ordered height, left, margin-left, margin-top, position, top, width